lib/rubocop/cop/graphql/unnecessary_argument_camelize.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module GraphQL
+      #  This cop checks if each argument has an unnecessary camelize.
+      #
+      # @example
+      #   # good
+      #
+      #   class UserType < BaseType
+      #     field :name, String, "Name of the user", null: true do
+      #       argument :filter, String, required: false, camelize: false
+      #     end
+      #   end
+      #
+      #   # good
+      #
+      #   class UserType < BaseType
+      #     argument :filter, String, required: false, camelize: false
+      #   end
+      #
+      #  # bad
+      #
+      #   class UserType < BaseType
+      #     argument :filter, String, required: false, camelize: false
+      #   end
```

Comment:
this "good" and "bad" example are the same. am i missing something?

Reply:
good point, should be `argument :filter, String, required: false` in the good section, will update master